app/models/importers/all_content_items.rb

```diff
@@ -1,10 +1,20 @@
 module Importers
   class AllContentItems
+    attr_accessor :metric_builder, :content_items_service
+
+    def initialize
+      @metric_builder = MetricBuilder.new
+      @content_items_service = ContentItemsService.new
+    end
+
     def run
-      content_item_importer = ContentItemsByOrganisation.new
+      document_types = Rails.configuration.document_types
```

Comment:
I would create `attr_writer` with the document_types so it can be updated in the test, and I would create a reader this way:

```ruby
def document_types
  @document_types ||= Rails.configuration.document_types
```